packages/drizzle/src/queries/getTableColumnFromPath.ts

```diff
@@ -53,6 +53,7 @@ type Args = {
    * If creating a new table name for arrays and blocks, this suffix should be appended to the table name
    */
   tableNameSuffix?: string
+  useAlias?: boolean
```

Comment:
We should be in the habbit of adding jsdocs to fields, but this is pretty straightforward.